pkg/ccl/backupccl/testdata/backup-restore/schedule-privileges

```diff
@@ -170,25 +159,4 @@ DROP SCHEDULE $testuserFullID;
 DROP SCHEDULE $testuserIncID;
 ----
 
-# But testuser can't drop, alter, resume or pause the root owned schedules.
```

Comment:
these seem like test cases we still want to cover. can you think of a way to adapt the test to cover them?